packages/cerebral/package.json

```diff
@@ -37,6 +37,7 @@
     ]
   },
   "devDependencies": {
-    "prop-types": "^15.5.10"
+    "prop-types": "15.5.10"
```

Comment:
@christianalfoni Here the caret can stay but you need to declare it a second time in monorepo root package.json without caret. Little confusing. 
BTW comma is missing here.